client/helpdata.cpp

```diff
@@ -1959,14 +1955,13 @@ char *helptext_unit(char *buf, size_t bufsz, struct player *pplayer,
       }
     }
     unit_class_iterate_end;
-    QString orlist = strvec_to_or_list(classes);
 
     cat_snprintf(buf, bufsz,
                  // TRANS: %s is a list of unit classes separated by "or".
                  PL_("* Can carry and refuel %d %s unit.\n",
                      "* Can carry and refuel up to %d %s units.\n",
                      utype->transport_capacity),
-                 utype->transport_capacity, qUtf8Printable(orlist));
+                 utype->transport_capacity, qUtf8Printable(strvec_to_or_list(classes)));
```

Comment:
This line is too long, you can't possibly have applied `clang-format` :wink:

Reply:
Thanks! I didnt :smile:  Will do!